docs/tutorials/getting_started.rst

```diff
@@ -0,0 +1,550 @@
+    ## by MPAS-Analysis in diagnostics computations.
+
+    # The base path to the diagnostics directory.  Typically, this will be a shared
+    # directory on each E3SM supported machine (see the example config files for
+    # its location).  For other machines, this would be the directory pointed to
+    # when running "download_analysis_data.py" to get the public observations,
+    # mapping files and region files.
+    baseDirectory = /path/to/diagnostics
+
+For ``baseDirectory``, supply the path where you downloaded the data
+``</path/to/mpas_analysis/diagnostics>``.
+
+4.4 [input]
+~~~~~~~~~~~
+
+The ``[input]`` section provides paths to the E3SM simulation data and the name
+of the MPAS-Ocean and MPAS-Seaice mesh.
+
+.. code-block:: ini
+
+    [input]
+    ## options related to reading in the results to be analyzed
+
+    # directory containing model results
+    baseDirectory = /dir/for/model/output
+
+    # Note: an absolute path can be supplied for any of these subdirectories.
+    # A relative path is assumed to be relative to baseDirectory.
+    # In this example, results are assumed to be in <baseDirecory>/run
+
+    # subdirectory containing restart files
+    runSubdirectory = run
+    # subdirectory for ocean history files
+    oceanHistorySubdirectory = run
+    # subdirectory for sea ice history files
+    seaIceHistorySubdirectory = run
+
+    # names of namelist and streams files, either a path relative to baseDirectory
+    # or an absolute path.
+    oceanNamelistFileName = run/mpaso_in
+    oceanStreamsFileName = run/streams.ocean
+    seaIceNamelistFileName = run/mpassi_in
+    seaIceStreamsFileName = run/streams.seaice
+
+    # names of ocean and sea ice meshes (e.g. oEC60to30, oQU240, oRRS30to10, etc.)
+    mpasMeshName = mesh
+
+The ``baseDirectory`` is the path where you untarred the example run.
+
+The ``mpasMeshName`` is the standard E3SM name for the MPAS-Ocean and
+MPAS-Seaice mesh.  In this example, this is ``oQU480``, meaning the
+quasi-uniform 480-km mesh for the ocean and sea ice.
+
+The ``runSubdirectory`` must contain valid MPAS-Ocean and MPAS-Seaice restart
+files, used to get information about the MPAS mesh and the ocean vertical grid.
+
+The ``oceanHistorySubdirectory`` must contain MPAS-Ocean monthly mean output
+files, typically named::
+
+   mpaso.hist.am.timeSeriesStatsMonthly.YYYY-MM-DD.nc
+
+Similarly, ``seaIceHistorySubdirectory`` contains the MPAS-Seaice monthly mean
+output::
+
+   mpassi.hist.am.timeSeriesStatsMonthly.YYYY-MM-DD.nc
+
+Finally, MPAS-Analysis needs a set of "namelists" and "streams" files that
+provide information on the E3SM configuration for MPAS-Ocean and MPAS-Seaice,
+and about the output files, respectively.  These are typically also found in
+the ``run`` directory.
+
+For the example data, only ``baseDirectory`` and ``mpasMeshName`` need to be
+set, the other options can be left as the defaults from the
+`example config file`_.
+
+4.5 [output]
```

Comment:
Hey @xylar: nice all-hands presentation, thank you for doing that!
I was thinking, maybe we can start spelling out something about the cached data more explicitly here, with location, etc, mentioning about possible customizations for specific plots not generated by mpas-analysis.

Reply:
I would suggest that as maybe a new documentation page or tutorial.  I would definitely welcome that addition!  Or I will try to get to it when I can.  Or we could work on it together over time.

Reply:
Yes, I agree that it needs a new addition in the documentation, but since this is going public soon, I wonder if we can start adding something here. I'll put down some text later today, and we can decide where to include it.

Reply:
Okay, sounds good. I don't think I would include things at that level in this particular tutorial but it would be wonderful to have the text ready. I want to keep this just focused on the basics.

Reply:
sounds good.